wgsl/index.bs

```diff
@@ -11737,8 +10999,861 @@ When calling a built-in function, all arguments to the function are evaluated
+      <td>`T` is [=scalar=]
+  <tr><td>Description
+      <td>[=Component-wise=] construction of a four-component [=vector=] with `e1`, `e2`, `e3`, and `e4` as components.
+</table>
+<table class='data builtin'>
+  <tr><td style="width:10%">Overload
+      <td><xmp highlight=rust>
+@const @must_use fn vec4<T>(e1 : T, v1 : vec2<T>, e2 : T) -> vec4<T>
+@const @must_use fn vec4(v1 : vec2<T>, v2 : vec2<T>) -> vec4<T>
+</xmp>
+  <tr><td>Parameterization
+      <td>`T` is [=scalar=]
+  <tr><td>Description
+      <td>[=Component-wise=] construction of a four-component [=vector=] with `e1`, `v1.x`, `v1.y`, and `e2` as components.
+</table>
+<table class='data builtin'>
+  <tr><td style="width:10%">Overload
+      <td><xmp highlight=rust>
+@const @must_use fn vec4<T>(e1 : T, e2 : T, v1 : vec2<T>) -> vec4<T>
+@const @must_use fn vec4(e1 : T, e2 : T, v1 : vec2<T>) -> vec4<T>
+</xmp>
+  <tr><td>Parameterization
+      <td>`T` is [=scalar=]
+  <tr><td>Description
+      <td>[=Component-wise=] construction of a four-component [=vector=] with `e1`, `e2`, `v1.x`, and `v1.x` as components.
+</table>
+<table class='data builtin'>
+  <tr><td style="width:10%">Overload
+      <td><xmp highlight=rust>
+@const @must_use fn vec4<T>(v1 : vec2<T>, v2 : vec2<T>) -> vec4<T>
+@const @must_use fn vec4(v1 : vec2<T>, v2 : vec2<T>) -> vec4<T>
+</xmp>
+  <tr><td>Parameterization
+      <td>`T` is [=scalar=]
+  <tr><td>Description
+      <td>[=Component-wise=] construction of a four-component [=vector=] with `v1.x`, `v1.y`, `v2.x`, and `v2.y` as components.
+</table>
+<table class='data builtin'>
+  <tr><td style="width:10%">Overload
+      <td><xmp highlight=rust>
+@const @must_use fn vec4<T>(v1 : vec2<T>, e1 : T, e2 : T) -> vec4<T>
+@const @must_use fn vec4(v1 : vec2<T>, e1 : T, e2 : T) -> vec4<T>
+</xmp>
+  <tr><td>Parameterization
+      <td>`T` is [=scalar=]
+  <tr><td>Description
+      <td>[=Component-wise=] construction of a four-component [=vector=] with `v1.x`, `v1.y`, `e1`, and `e2` as components.
+</table>
+<table class='data builtin'>
+  <tr><td style="width:10%">Overload
+      <td><xmp highlight=rust>
+@const @must_use fn vec4<T>(v1 : vec3<T>, e1 : T) -> vec4<T>
+@const @must_use fn vec4(v1 : vec3<T>, e1 : T) -> vec4<T>
+</xmp>
+  <tr><td>Parameterization
+      <td>`T` is [=scalar=]
+  <tr><td>Description
+      <td>[=Component-wise=] construction of a four-component [=vector=] with `v1.x`, `v1.y`, `v1.z`, and `e1` as components.
+</table>
+<table class='data builtin'>
+  <tr><td style="width:10%">Overload
+      <td><xmp highlight=rust>
+@const @must_use fn vec4<T>(e1 : T, v1 : vec3<T>) -> vec4<T>
+@const @must_use fn vec4(e1 : T, v1 : vec3<T>) -> vec4<T>
+</xmp>
+  <tr><td>Parameterization
+      <td>`T` is [=scalar=]
+  <tr><td>Description
+      <td>[=Component-wise=] construction of a four-component [=vector=] with `e1`, `v1.x`, `v1.y`, and `v1.z` as components.
+</table>
+
+## Value Reinterpretation Built-in Functions ## {#value-reinterp-builtin-functions}
```

Comment:
The "Value" part here is too suggestive of value conversion.

Suggest "Representation Reinterpretation Built-in Functions" or "Bit Reinterpretation Built-in Functions"